docker-compose.prod.yml

```diff
@@ -2,11 +2,11 @@ include:
   - docker-compose.yml
 
 services:
-  backup:
-    build: "./backup"
-    networks:
-      mysql-network:
```

Comment:
doesn't it need access to the mysql network?

Reply:
backup and all works fine

Reply:
Ah no. It does need the network

Reply:
@harshkhandeparkar fixed